governance/src/agent.spec.js

```diff
@@ -0,0 +1,564 @@
+const {
+  Finding, createTransactionEvent, TransactionEvent, ethers,
+} = require('forta-agent');
+
+const { provideHandleTransaction, provideInitialize } = require('./agent');
+const { getAbi } = require('./utils');
+const { createMockEventLogs, getObjectsFromAbi } = require('./test-utils');
+
+const config = require('../bot-config.json');
+
+/**
+ * TransactionEvent(type, network, transaction, traces, addresses, block, logs, contractAddress)
+ */
+function createTxEvent({ logs, addresses }) {
+  return new TransactionEvent(null, null, null, [], {}, null, logs, addresses);
+}
+
+// check the configuration file to verify the values
+describe('check bot configuration file', () => {
+  it('procotolName key required', () => {
```

Comment:
```suggestion
  it('protocolName key required', () => {
```
Probably pasted in from some other typo code.